File: GreenTicket-WebAPI/Models/EventPageDto.cs
using static System.Net.Mime.MediaTypeNames;

namespace GreenTicket_WebAPI.Models
{
    public class EventPageDto
    {
        public EventPageDto()
        {
            Sections = new HashSet<SectionDto>();
        }

        public int Id { get; set; }
        public string Name { get; set; } = null!;
        public DateTime StartDateTime { get; set; }
        public DateTime EndDateTime { get; set; }
        public string Description { get; set; } = null!;
        public string City { get; set; } = null!;
        public string VenueName { get; set; } = null!;
        public int LimitPerUser { get; set; }
        public string? EventCardImage { get; set; }
        public string? SeatingPlanImage { get; set; }
        public decimal PriceFrom { get; set; }
        public ICollection<SectionDto> Sections { get; set; }

    }
}

File: GreenTicket-ClientApp/src/app/models/eventPageDto.ts
import { SectionDto } from "./sectionDto";

export interface EventPageDto {
    id: number;
    name: string;
    startDateTime: Date;
    endDateTime: Date;
    description: string;
    city: string;
    venueName: string;
    eventCardImage: string;
    seatingPlanImage: string;
    priceFrom: number;
    limitPerUser: number;
    sections: SectionDto[];

}